### PR TITLE
PE-3926: Turbo profile card plus dev tools

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/36vf894gea26o
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/75v6umsu1ptmo

### CODE REVIEW

File: assets/config/prod.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "defaultArweaveGatewayUrl": "https://arweave.net",
-  "useTurbo": true,
-  "defaultTurboUrl": "https://upload.ardrive.io",
+  "useTurboUpload": true,
+  "useTurboPayment": false,
```
Comment (matibat) [inline assets/config/prod.json:4]: Turbo uploads are enabled, Turbo payment is disabled 👍

File: dart_test.yaml [hunk 2]
```diff
@@ -4,3 +4,6 @@ include: dart_test_base.yaml
 # Save web results on JSON in a specific folder
 file_reporters:
   json: reports/web/tests.json
+
+tags:
+  golden:
```
Comment (matibat) [inline dart_test.yaml:9]: Hmmm what's this gonna do?

File: flake.nix
Comment (matibat) [inline flake.nix:1]: What a strange syntax! Double single-quote for multi-line strings? 😵‍💫

File: ios/Runner/Info.plist [hunk 3]
```diff
@@ -81,5 +81,7 @@
 	<true/>
 	<key>UIFileSharingEnabled</key>
 	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
```
Comment (matibat) [inline ios/Runner/Info.plist:85]: It's not clear to me why do we need this

File: lib/blocs/turbo_balance/turbo_balance_cubit.dart [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      emit(TurboBalanceLoading());
+
+      final balance = await paymentService.getBalance(wallet: wallet);
+
+      emit(TurboBalanceSuccessState(balance: balance));
+    } catch (e) {
+      logger.e('Error getting balance: $e');
+
+      /// Wait for the animation to finish
+      await Future.delayed(const Duration(milliseconds: 500));
```
Comment (matibat) [inline lib/blocs/turbo_balance/turbo_balance_cubit.dart:30]: I'm not fully aware of this part so I have to ask: is the animation fixed to 500ms after a failure? So, will this delay match exactly the timing of the animation?

Comment (matibat) [reply lib/blocs/turbo_balance/turbo_balance_cubit.dart:30]: What will it happen if we didn't do this?

File: lib/components/top_up_dialog.dart [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  @override
+  State<TopUpDialog> createState() => _TopUpDialogState();
+}
+
+class _TopUpDialogState extends State<TopUpDialog> {
+  @override
+  Widget build(BuildContext context) {
+    return ArDriveStandardModal(
+      title: 'turbo',
```
Comment (matibat) [inline lib/components/top_up_dialog.dart:15]: Needs to be localized?

File: lib/dev_tools/app_dev_tools.dart
Comment (matibat) [inline lib/dev_tools/app_dev_tools.dart:1]: I don't think the text in here will be put in the localization files

File: lib/utils/winston_to_ar.dart
Comment (matibat) [inline lib/utils/winston_to_ar.dart:1]: Do we have any unit tests for this method?

**Review by matibat — APPROVED** [review]
LGTM overall - Most of my comments are just questions :)

File: dart_test.yaml (hunk 2 above)
Comment (agsuy) [reply dart_test.yaml:9]: IIRC those are some tests from Javed. I am still not sure we want to continue implementing tests that way.

**Pull request merged**